### PR TITLE
Create P6_G2_Schmutz_Herrero/

### PR DESCRIPTION
**Pull request by Marioherreroglez**

### CODE REVIEW

**Comment by gmaze** [comment]
your #1 is empty, please add some files (see #2 procedure)

**Comment by gmaze** [comment]
@Marioherreroglez would be better to create a folder name "P6_G2_Schmutz_Herrero" under "projects" and to place there all your files

**Comment by Marioherreroglez** [comment]
@gmaze Normally I should have created a folder named P6 - Schmutz and Herrero in my branch. I have afterwards pull requested the changes. Have I done it correctly?

File: project/Project 6 -Schmutz and Herrero/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (gmaze) [inline project/Project 6 -Schmutz and Herrero/README.md:1]: Why is your README file empty ?

**Review by gmaze — COMMENTED** [review]
Almost there:
- [ ] You should delete the empty file ``P6_G2_Schmutz_Herrero``
- [ ] You should add minimal content to your README.md file

**Pull request merged**